migrations/2_deploy_contracts.ts

```diff
@@ -1,6 +1,8 @@
 var RockPaperScissorsGame = artifacts.require("./RockPaperScissorsGame.sol");
```

Comment:
I'm not sure if we want to manually change the migration file but since `fmg` is separated into packages all their migration files use the same number/name.